### PR TITLE
Move to github actions

### PR DESCRIPTION
**Pull request by MancunianSam**
- Move to GitHub actions.
- Remove temporary push triggers
- Add fake AWS credentials
- Use the new version of aws utils
- Run postgres with the tests
- Test the build and deploy jobs
- Use the correct naem
- Remove temporary branch code
- Use updated lambda name
- Run jobs in sequence.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: TDR Run Flyway Test
```
Comment (TomJKing) [inline .github/workflows/test.yml:1]: Is there a reason this is called `TDR Run Flyway Test`?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one minor question.

File: .github/workflows/test.yml (hunk 1 above)
Comment (MancunianSam) [reply .github/workflows/test.yml]: Because I copied it of the consignment-api-data repo and didn't change it :smile: I've renamed it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**